web/client/components/map/leaflet/PopupSupport.jsx

```diff
@@ -38,6 +38,11 @@ export default class PopupSupport extends React.Component {
         popups: [],
         onPopupClose: () => {}
     }
+    componentWillMount() {
+        // This prevent the pointermove to be sent event when stopevent is active.
+        document.querySelector('.leaflet-popup-pane').addEventListener('mousemove', this.stopPropagationOnMouseMove);
```

Comment:
Maybe, if possible, is good to not query the whole document but only the div of the map, with the query selector. Same for openlayers

Reply:
Done